packages/analytics/analytics-utilities/package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "dev": "cross-env USE_SANDBOX=true vite",
     "build": "run-s typecheck build:package build:types",
-    "build:docs": "typedoc --plugin typedoc-plugin-markdown --out docs src/index.ts --entryDocument analytics-utils.md --gitRevision main --githubPages false --readme none --tsconfig ./tsconfig.build.json",
+    "build:docs": "typedoc --plugin typedoc-plugin-markdown --out docs src/index.ts --gitRevision main --githubPages false --readme none --tsconfig ./tsconfig.build.json",
```

Comment:
Rather than a pre-commit hook, could you just add the `build:docs` script at the end of the `build` script above?